Makefile

```diff
@@ -10,8 +10,15 @@ DOCKER_IMAGE_NAME = quay.io/reconmap/web-client
 DOCKER_CONTAINER_NAME = reconmap-web-client
 DOCKER_DEV_TAG = reconmap/web-client:dev
 
-HOST_UID=$(shell id -u)
-HOST_GID=$(shell id -g)
+# macOS is using different IDs than linux
+UNAME=$(shell uname)
```

Comment:
I would suggest to remove this completely and keep just the HOST_UID=1000 and HOST_GID=1000.
If I understand your intention correctly, we just want to know ids of the created user inside the docker. So, if we simply use the hardcoded value, it should work. 
I assume that the original `shell id -u` also won't work on Windows machine.

But I did not want to remove it yet, because I want to double-check that the removal won't break anything. It is possible that I do not see the whole picture.

Reply:
The reason why it is dynamically obtaining user and group ids is that users on the host and container need to match so that files in volumes are created with the same permissions. Otherwise you end up in situations where files created in the container are owned by root or another user in the host for which you don't have access. This setup seems to work ok locally *and* on the Github actions.
Hope the above makes sense. Is the `shell id` causing you any trouble?

Reply:
Yeah on macOS the uid is 501 and group IDs are even more different (the one which was used by the makefile is 20 and it already exists inside the container) even more it seems that the user does not have the group with the same name and id as in linux. So the groupadd command was failing. I've changed it to 1000:1000 and it seems to be working just fine

Reply:
Happy to merge this change to the Makefile because it's only affecting Macos so won't cause Linux users or the pipelines any trouble.